projects/melon-types/AsyncLoop.d.ts

```diff
@@ -0,0 +1,6 @@
+declare class AsyncLoop<T> {
+    constructor(action: Function, times?: number, delay?: number, errorAction?: Function);
+    stop: () => void;
+    times: number;
+    execute: () => Promise<T>;
+}
```

Comment:
Any news on the effort of the TypeScript build being directly inside `melon-runtime`? I don't see any issues with it being a pre-build step which generates `.js` before we do anything.
If it's truly not something viable right now, an improvement for better TypeScript integration would be to have these `d.ts` files inside `melon-runtime` somehow.
I know we're tied to .NET's JS shenanigans but just having them as "extra files" living together with JS that are copied over would already be nice. Only thing that matters is them being a folder that package.json's typings direct to.

Reply:
Not really